trestle/core/markdown/md_writer.py

```diff
@@ -162,3 +168,17 @@ def get_lines(self) -> List[str]:
     def get_text(self) -> str:
         """Get the text as currently written."""
         return '\n'.join(self._lines)
+
+    def _add_subnode_text(self, node: MarkdownNode) -> None:
+        """Add the header text to the markdown contents."""
+        for line in node.content.raw_text.split('\n'):
+            self.new_line(line)
+        for subnode in node.subnodes:
+            self._add_subnode_text(subnode)
```

Comment:
This will duplicate things. raw_text of a parent node already contains markdown of children nodes...

Reply:
reverted the change